docs/config-description.md

```diff
@@ -1,452 +1,600 @@
 - **Required**: True
 - **Type**: int
-- **Default value**: 20
-- **Constraints**: The minimum value is `0`. The maximum value is `65535`
- -   **Description**: Batch size for write operations in Cassandra.
-### Key: database.cassandra.connect_timeout
+- **Default value**: `20`
+- **Constraints**: The minimum value is `0`. The maximum value is `65535`.
+- **Description**: Represents the batch size for write operations in Cassandra.
+
+### database.cassandra.connect_timeout
+
 - **Required**: False
 - **Type**: int
 - **Default value**: None
-- **Constraints**: The minimum value is `0`. The maximum value is `4294967295`
- -   **Description**: The maximum amount of time in seconds the system will wait for a connection to be successfully established with the database.
-### Key: database.cassandra.request_timeout
+- **Constraints**: The minimum value is `0`. The maximum value is `4294967295`.
+- **Description**: The maximum amount of time in seconds that the system waits for a database connection to be established.
+
+### database.cassandra.request_timeout
+
 - **Required**: False
 - **Type**: int
 - **Default value**: None
-- **Constraints**: The minimum value is `0`. The maximum value is `4294967295`
- -   **Description**: The maximum amount of time in seconds the system will wait for a request to be fetched from database.
-### Key: database.cassandra.username
+- **Constraints**: The minimum value is `0`. The maximum value is `4294967295`.
+- **Description**: The maximum amount of time in seconds that the system waits for a request to be fetched from the database.
+
+### database.cassandra.username
+
 - **Required**: False
 - **Type**: string
 - **Default value**: None
 - **Constraints**: None
- -   **Description**: The username used for authenticating with the database.
-### Key: database.cassandra.password
+- **Description**: The username used for authenticating with the database.
+
+### database.cassandra.password
+
 - **Required**: False
 - **Type**: string
 - **Default value**: None
 - **Constraints**: None
- -   **Description**: The password used for authenticating with the database.
-### Key: database.cassandra.certfile
+- **Description**: The password used for authenticating with the database.
+
+### database.cassandra.certfile
+
 - **Required**: False
 - **Type**: string
 - **Default value**: None
 - **Constraints**: None
- -   **Description**: The path to the SSL/TLS certificate file used to establish a secure connection between the client and the Cassandra database.
-### Key: allow_no_etl
+- **Description**: The path to the SSL/TLS certificate file used to establish a secure connection between the client and the Cassandra database.
+
+### allow_no_etl
+
 - **Required**: True
 - **Type**: boolean
-- **Default value**: True
+- **Default value**: `True`
 - **Constraints**: None
- -   **Description**: If True, no ETL nodes will run with Clio.
-### Key: etl_sources.[].ip
+- **Description**: If set to `True`, no ETL nodes will run with Clio.
```

Comment:
This sounds wrong. This settings merely allows Clio to start without any ETL sources (rippled nodes) being available at startup.